style.css

```
@@ -0,0 +1,38 @@
+head {
+    font-size: larger;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    color:darkgreen
+}
+h1 {
+    font-size:60px;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    color:rgb(3, 61, 3)
```

Try to keep an attention to detail of code formatting (spacing, semi-colons, etc) so that it is consistent

```suggestion
    color: rgb(3, 61, 3);
```